components/sign-out-button.jsx

```diff
@@ -24,7 +24,7 @@ class SignOutButton extends React.Component {
 
   render() {
     return (
-      <button className="btn btn-link inline-link" onClick={(event) => this.handleLogOutBtnClick(event)}>Sign out</button>
+      <button className="btn-signout" onClick={(event) => this.handleLogOutBtnClick(event)}>Sign out</button>
```

Comment:
Any reason not using the existing classes `btn-link` and `inline-link`?

Reply:
no specific reason, tbh forgot those classes existed 😅 ...updating now